opennlp-tools/src/main/java/opennlp/tools/namefind/NameFinderME.java

```diff
@@ -219,6 +220,11 @@ public void probs(double[] probs) {
   public static TokenNameFinderModel train(String languageCode, String type,
           ObjectStream<NameSample> samples, TrainingParameters trainParams,
           TokenNameFinderFactory factory) throws IOException {
+
+    trainParams.putIfAbsent(TrainingParameters.ALGORITHM_PARAM, PerceptronTrainer.PERCEPTRON_VALUE);
+    trainParams.putIfAbsent(TrainingParameters.CUTOFF_PARAM, "0");
+    trainParams.putIfAbsent(TrainingParameters.ITERATIONS_PARAM, Integer.toString(300));
```

Comment:
Just curious if there's any reason for `Integer.toString()` instead of `"300"` (like the `"0"` on the line above).